.github/workflows/markdown-link-check-config.json

```diff
@@ -11,6 +11,9 @@
     },
     {
       "pattern": ".*❓.*"
+    },
+    {
+      "pattern": "^https:\/\/www\\.microsoft\\.com"
```

Comment:
We are getting a failure on this URL: https://www.microsoft.com/msrc/faqs-report-an-issue
It works but the tester seems to think otherwise.